release/Configure

```diff
@@ -20,7 +20,15 @@ fi
 ### Ask config questions
 if [ -z "$MOLOCH_INTERFACE" ]; then
     echo -n "Found interfaces: "
-    /sbin/ifconfig | grep "^[a-z]" | cut -d: -f1 | cut -d" " -f1 | paste -s -d";"
+    if [ -f /etc/redhat-release ];
```

Comment:
instead of checking for redhat should it just check for ifconfig so it might work on any OS without ifconfig?